app/controllers/sites_controller.rb

```diff
@@ -57,11 +54,21 @@ def insights
     binned_data << unknown_group if unknown_group
     @document_years = binned_data
     # Create table data.
-    default_group = Document::STATUSES.map { |status| [status, 0] }.to_h
+    default_group = Document::DECISION_TYPES.keys.map { |status| [status, 0] }.to_h
     @category_groups = {}
-    @documents.group([:document_category, :status]).count.each do |groups, group_count|
+    @documents.group([:document_category, :accessibility_recommendation]).count.each do |groups, group_count|
       @category_groups[groups[0]] = default_group.clone if @category_groups[groups[0]].nil?
-      @category_groups[groups[0]][groups[1]] = group_count
+      if Document::DECISION_TYPES.keys.exclude? groups[1]
```

Comment:
Can you explain what's happening here? Is this used for creating the counts in each decision category and displaying these on the left top sidebar?

Reply:
Got it! Thank you